packages/react-scripts/config/css.html

```diff
@@ -0,0 +1,5 @@
+<% _.each(htmlWebpackPlugin.files.css, file => { %><link
+    rel="stylesheet"
+    href="<%= file %>"
+  />
+  <% }); %>
```

Comment:
```suggestion
  <% }); %>
  
```

Restore empty line in the end of file.